pennylane/operation.py

```diff
@@ -1192,7 +1207,7 @@ def parameter_frequencies(self):
         if self.num_params == 1:
             # if the operator has a single parameter, we can query the
             # generator, and if defined, use its eigenvalues.
-            gen = self.generator()
+            gen = self.generator()  # pylint: disable=assignment-from-no-return
```

Comment:
🤔

Reply:
This one turned out to be superfluous. :D